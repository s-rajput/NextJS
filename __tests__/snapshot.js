// __tests__/snapshot.js
import React from 'react'
import renderer from 'react-test-renderer'
import About from '../pages/about-us'

it('renders aboutpage unchanged', () => {
  const tree = renderer.create(<About />).toJSON()
  expect(tree).toMatchSnapshot()
})