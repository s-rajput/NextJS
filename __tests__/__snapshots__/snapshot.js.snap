// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders aboutpage unchanged 1`] = `
<h1>
  This is about us page
</h1>
`;

exports[`renders homepage unchanged 1`] = `
<h1>
  This is about us page
</h1>
`;
